package.json

```diff
@@ -156,6 +156,8 @@
   },
   "resolutions": {
     "dgeni-packages/typescript": "3.7.4",
+    "@bazel/jasmine/jasmine": "3.5.0",
+    "@bazel/jasmine/jasmine-core": "3.5.0",
```

Comment:
These are temporarily until `@bazel/jasmine` has been updated to jasmine v3.5.0. I created a ticket for tracking: DEV-37, and also sent a PR upstream to update jasmine.